agent_build/linux/prepare_build_environment.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -e
+
+yum install -y tar wget perl
+yum groupinstall -y 'Development Tools'
+
+# Build and install openssl, since python 3.8+ requires newer version than centos 7 can provide.
+wget https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1k.tar.gz -O /tmp/openssl-build.tar.gz
```

Comment:
Do we need to keep this up to date when a new version of OpenSSL is released (aka is openssl statically linked in our "binary" or not)?

Reply:
It has to be bundled with Python, so that means that it is our responsibility to keep it up to date.